lib/skeleton/config/scripts/android/run_tests_all_devices.sh

```diff
@@ -5,10 +5,16 @@
 #
 # $1 -> parameter with the name of the apk
 # $2 -> parameter to indicates the tapume to run. Can be null and can have other 2 values: must or should
+# $3 -> parameter with tests specs. For exxameple, put "feature/<feature_name>" to select an specific feature file; put "" for everything
 
 ## CODE BEGIN  #############################################################
 [ $# -lt 1 ] && echo "Wrong number of parameters." && exit 1
 
+#setting which enviroment we are dealing wiff (CI or local)
+enviroment='CI'
```

Comment:
Can you explain more about this variable? I can't figure out its purpose.
Typo in enviroment (environment)